site/en/workflows.md

```diff
@@ -778,6 +893,56 @@ This processor can be used to analyse the output of the OCR.
   </tbody>
 </table>
 
+## Format Conversion (Optional)
+
+OCR-D produces PAGE XML files which contain the recognized text as well as detailed
+information on the structure of the processed pages, the coordinates of the recognized
+elements etc. Optionally, the PAGE XML can be converted to a different output format.
+
+### Step 18: Format Conversion
+
+In this processing step the produced PAGE XML files can be converted to
+ALTO, hOCR or text files. Note that ALTO and hOCR can also be converted into
```

Comment:
or PDF! https://github.com/JKamlah/ocrd_pagetopdf is working quite well already, and can be easily _viewed_.

Reply:
https://github.com/OCR-D/ocrd-website/pull/86/commits/d523926ba3c912179c05cbfa6ee747b11a3c1845